config.yaml

```diff
@@ -10,7 +10,7 @@ options:
     description: Port used by MySQL
   admin-password:
     type: string
-    default: ''
+    default: 'Password'
```

Comment:
A default password in a github repository, without a mechanism to force the user to change it on first deployment, is a bit worrying. Is there no other way to allow the administrator to set a default password, for example an environment variable ?

Reply:
Thanks Balbir, I forget to mention that this PR was only to move forward with the development. I agree with you!
I will follow this in #22